gatsby_fp/src/components/fieldComponents/IntellCont.js

```diff
@@ -1,112 +1,37 @@
 import React from "react"
+import SubHeading from "./intellContComponents/SubHeading.js"
 const shortid = require("shortid")
 
 export default function IntellCont({ intellContList }) {
+
+  function findExistingContypes(intellContList){
+    var contypeList = []
+    intellContList.forEach(element => {
+      if(element.contype && !(contypeList.includes(element.contype))){
+        if(element.contype !== "Other"){
```

Comment:
To allow for the possibility that the string "Other" becomes something else in the future (like "Other (please specify)" we should avoid using the string in our logic. Instead, we can just go ahead and `push` the `contype` if `contypeother` does not exist for this same element. If it turns out that for some reason the DM interface actually lets people populate `contypeother` even when `contype` isn't "Other", then we can go back to this as a last resort.